cmd/src/campaigns_common.go

```diff
@@ -331,6 +330,12 @@ func campaignsParseSpec(out *output.Output, svc *campaigns.Service, input io.Rea
 // printExecutionError is used to print the possible error returned by
 // campaignsExecute.
 func printExecutionError(out *output.Output, err error) {
+	// exitCodeError shouldn't generate any specific output, since it indicates
+	// that this was done deeper in the call stack.
+	if _, ok := err.(*exitCodeError); ok {
```

Comment:
Hmm, are you sure about this? I'm asking because I vaguely remember something related to killed-by-interrupt and `exitCodeError` but I am not sure. If you're sure, though, and this still works when and reports errors correctly when killing it with Ctrl-C, then that's okay.

Reply:
In response to this, I tested that specifically, and it does still report errors as expected with Ctrl-C.